sepolicy/vendor/file_contexts

```diff
@@ -49,6 +49,7 @@
 /vendor/bin/hw/android.hardware.gatekeeper@1.0-service.optee                    u:object_r:hal_gatekeeper_optee_exec:s0
 /vendor/bin/hw/android.hardware.sensors@1.0-service.xenvm                       u:object_r:hal_sensors_default_exec:s0
 /vendor/bin/hw/android.hardware.health@2.0-service.xenvm                        u:object_r:hal_health_default_exec:s0
+/vendor/bin/hw/android.hardware.boot@1.0-service.xenvm				u:object_r:hal_bootctl_default_exec:s0
```

Comment:
May be it's better to use spaces instead of tabs to keep alignment of text.

Reply:
Done